css-index.css

```diff
@@ -0,0 +1,76 @@
+@media (min-width:900px){
+  .container{
+    width: auto;
+    border: 3px solid black;
```

Comment:
pregunta, por un border negro si tu imagen de background es del mismo color? jeje

Reply:
aaaa sorry I forgot to remove that it was only to distinguish that it was working correctly

Reply:
aa dont worry, I understand you

Reply:
At the moment I occupy them to limit the spaces of my document